reflex-algos/src/main/python/mlcomp/test/pipelines/simple-pipeline.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Simple MCenter runner test",
+    "engineType": "Python",
+    "systemConfig": {
+        "statsDBHost": "localhost",
+        "statsDBPort": 8086,
+        "mlObjectSocketSinkPort": 7777,
+        "mlObjectSocketSourcePort": 1,
+        "workflowInstanceId": "8117aced55d7427e8cb3d9b82e4e26ac",
```

Comment:
Yes. The pipeline is supposed to include the system config part, although the only fields that are expected to be there, depending on the component's tagged attributes are: `modelFileSinkPath` and `modelFileSourcePath`.
So, basically, if the given pipeline does not include an input / output model paths, then it can be dropped. 
We need to cleanup this part in the system. 
Specifically, for this example I can remove it.